### PR TITLE
[next] Graphics memory leak

### PR DESCRIPTION
**Pull request by ivanpopelyshev**
PR contains only simple fixes

Mesh refactoring will be separate

> `this.textureIds.length = 0;`

That thing crashed https://pixijs.io/examples/?v=next#/basics/textured-mesh.js

> `elements[i] = null;`

Backport from v4 fix

### CODE REVIEW

File: packages/mesh/src/Mesh.js [hunk 1]
```diff
@@ -107,6 +107,8 @@ export default class Mesh extends Container
          */
         this.vertexDirty = 0;
 
+        this._transformID = -1;
```
Comment (cursedcoder) [inline packages/mesh/src/Mesh.js]: btw why -1?

Comment (ivanpopelyshev) [reply packages/mesh/src/Mesh.js]: Same as in `Sprite`. 

`transform.worldID` is always non-negative.

**Comment by ivanpopelyshev** [comment]
Pushed for https://pixijs.io/examples/?v=next#/basics/spritesheet.js

File: packages/graphics/src/GraphicsGeometry.js [hunk 2]
```diff
@@ -215,9 +215,11 @@ export default class GraphicsGeometry extends BatchGeometry
             this.colors.length = 0;
             this.uvs.length = 0;
             this.indices.length = 0;
+            this.textureIds.length = 0;
 
-            for (let i = 0; i < DRAW_CALL_POOL.length; i++)
+            for (let i = 0; i < this.drawCalls.length; i++)
```
Comment (cursedcoder) [inline packages/graphics/src/GraphicsGeometry.js]: so we have global `DRAW_CALL_POOL` and local `this.drawCalls`, would be nice to add some doc what is the purpose for duplicated `DRAW_CALL_POOL`

**Comment by bigtimebuddy** [comment]
Thanks @ivanpopelyshev! Great cleanup!

**Pull request merged**